src/acq/channel.c

```diff
@@ -226,7 +226,7 @@ void bl_acq_channel_enable(enum bl_acq_source source)
 			BL_MSG_SAMPLE_DATA32 : BL_MSG_SAMPLE_DATA16;
```

Comment:
Typo in commit title, please change to "field".

Reply:
+1